assets/models/system/Mistral-7B-Instruct-v0-1/spec.yaml

```diff
@@ -4,7 +4,7 @@ path: ./
 properties:
   SharedComputeCapacityEnabled: true
   SHA: 3dc28cf29d2edd31a0a7b8f0b21637059815b4d5
-  inference-min-sku-spec: 12|2|224|672
+  inference-min-sku-spec: 12|1|220|64
```

Comment:
lets check if model size would allow to be deployed on this spec? Please add deployment valdn for this one

Reply:
Model size is ~30GB . Disk space should not be a problem